lib/api/core/entryPoints/kuzzleProxy.js

```diff
@@ -1,48 +1,69 @@
+/*
+ * Kuzzle, a backend software, self-hostable and ready to use
+ * to power modern apps
+ *
+ * Copyright 2017 Kuzzle
+ * mailto: support AT kuzzle.io
+ * website: http://kuzzle.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 'use strict';
 
-var
+const
   Request = require('kuzzle-common-objects').Request,
   RequestContext = require('kuzzle-common-objects').models.RequestContext;
 
 /**
  * Manage communication with the load balancer
  *
+ * @class KuzzleProxy
  * @param {Kuzzle} kuzzle
- * @constructor
  */
-function KuzzleProxy (kuzzle) {
-  this.kuzzle = kuzzle;
-  this.channels = {};
-
-  return this;
-}
+class KuzzleProxy {
+  constructor(kuzzle) {
+    this.kuzzle = kuzzle;
+    this.channels = {};
+  }
 
-KuzzleProxy.prototype.init = function kuzzleProxyInit () {
-  this.kuzzle.services.list.proxyBroker.listen('request', onRequest.bind(this));
-  this.kuzzle.services.list.proxyBroker.listen('connection', onConnection.bind(this));
-  this.kuzzle.services.list.proxyBroker.listen('disconnect', onDisconnect.bind(this));
-  this.kuzzle.services.list.proxyBroker.listen('error', onDisconnect.bind(this));
-  this.kuzzle.services.list.proxyBroker.listen('httpRequest', onHttpRequest.bind(this));
-};
+  init() {
+    this.kuzzle.services.list.proxyBroker.listen('request', onRequest.bind(this));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 82 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)